### DISCUSSION TITLE
评论审核和违禁词过滤开启无效

### DISCUSSION THREAD

**Discussion by Legend-cpu** [walinejs/waline#900] [Q&A] [answered]
我在vercel上部署的waline，然后设置了环境变量COMMENT_AUDIT为true之后用户仍然可以直接不经过审核直接发表评论，并且我也不带build cache重新部署（redeploy）了并promote to production了，仍然没有效果。同时设置的IPQPS倒是生效了，而且不需要redeploy就能生效。
另外加的违禁词过滤也没有生效：
![image](https://user-images.githubusercontent.com/80209598/162228075-c414aa50-1a44-440a-891b-6a9c623c2002.png)
请问有办法解决这两个问题么？

**Comment by lizheming** [comment, marked as answer]
没有生效就是没有部署到环境，不需要 redeploy 就能生效这个是不可能的，可以再重新 check 一下流程。

**Reply by Legend-cpu** [reply]
但是我已经是redeploy了啊，感觉没有什么能做的了。。你们的waline评论审核是可以用的吗？

**Reply by lizheming** [reply]
可以的

**Reply by Legend-cpu** [reply]
哦哦我知道为什么了，我自己网站的waline配置里的那个waline server url用的网址指向的是第一次部署的时候生成的一个域名，我把它更新成我现在的vercel.app域名了，然后现在就都能用了。

**Reply by Mister-Hope** [reply]
。。。。。
